tests/installation/multipath.pm

```diff
@@ -19,7 +19,19 @@ use testapi;
 sub run {
     # Sometimes multipath detection takes longer
     assert_screen "enable-multipath", 60;
-    send_key "alt-y";
+    # This module is supposed to be invoked only if MULTIPATH is true
+    # or MULTIPATH_DIALOG_YES is set
+    if (get_var("MULTIPATH") or get_var("MULTIPATH_DIALOG_YES")) {
+        send_key "alt-y";
+    }
+    else {
+        # MULTIPATH_DIALOG_YES set to false
```

Comment:
In which situation we will get to this step? I suppose never, if we don't set `MULTIPATH_DIALOG_YES` or `MULTIPATH` , module `multipath.pm` will not be loaded.

Reply:
Set MULTIPATH_DIALOG_YES = 0, and you will get this case. See also the verification runs provided above.

Reply:
What is the point of having multipath device over iscsi available but not enable multipath?

Reply:
not enable multipath here != not enable multipath at all. One might want to do so only later,
e.g. after providing a customized multipath.conf.
This case is mentioned at the beginning of mwilck's comment 3 in [poo#46463](https://progress.opensuse.org/issues/46463).
Replying "yes" in this early multipath dialogue used to trigger a rather "greedy" action: multipathd and the device mapper would grab *all* disks (might be different in the very newest products: I'd have to check). This may well be unwanted, in particular if the system disk is not multipathed.